models/src/main/protobuf/CasperMessage.proto

```diff
@@ -27,15 +27,23 @@ service DeployService {
   rpc listenForContinuationAtName(Channels) returns (ListeningNameContinuationResponse) {}
 }
 
+message PhloLimit {
+  int64 value = 1;
+}
+
+message PhloPrice {
+  int64 value = 1;
+}
```

Comment:
what's the difference between PhloPrice and PhloLimit? shouldn't they be the same type?

Reply:
No, they are different things.